co_sim_io/impl/communication/file_communication.hpp

```diff
@@ -220,23 +221,28 @@ class FileCommunication : public Communication
             input_file >> rData[i];
         }
 
+        input_file.close();
```

Comment:
note to self, this is necessary for windows => file needs to be closed before removing it
Not sure why/how it was working before